README.md

```diff
@@ -63,12 +67,12 @@ Can be found in [RELEASE_NOTES](RELEASE_NOTES.md).
 <dependency>
     <groupId>com.github.romankh3</groupId>
     <artifactId>image-comparison</artifactId>
-    <version>4.0.0</version>
+    <version>4.0.1</version>
 </dependency>
 ```
 #### Gradle
 ```groovy
-compile 'com.github.romankh3:image-comparison:4.0.0'
+compile 'com.github.romankh3:image-comparison:4.0.1'
```

Comment:
Do you also need to update the version here?

Reply:
Of course, I'll do it. Thanks for your catch.